packages/reducers/src/makeReducer.test.js

```diff
@@ -29,4 +31,15 @@ describe('makeReducer', () => {
 		const reducer = makeReducer([['ADD', (state, { payload }) => state + payload]]);
 		expect(reducer(5, { type: 'ADD', payload: 3 })).toBe(8);
 	});
+
+	it('handles array as an action type', () => {
+		const reducer = makeReducer([[['TEST', 'SECOND'], () => 'ok', () => 'nope']]);
+		expect(reducer('something', { type: 'TEST' })).toBe('ok');
+		expect(reducer('something', { type: 'SECOND' })).toBe('ok');
+	});
+
+	it('handles function as an action type', () => {
+		const reducer = makeReducer([[equals('TEST'), () => 'ok', () => 'nope']]);
+		expect(reducer('something', { type: 'TEST' })).toBe('ok');
```

Comment:
please add unhappy path tests too, `makeReducer` expects initial state to be provided as the second argument. if you pass unknown type, the initial state should be returned